src/content/blog/astro-5150.mdx

```diff
@@ -0,0 +1,155 @@
+            assetQueryParams: process.env.DEPLOY_ID
+              ? new URLSearchParams({ deployId: process.env.DEPLOY_ID })
+              : undefined,
+          },
+        });
+      },
+    },
+  };
+}
+```
+
+The Vercel adapter now also benefits from these new APIs. Previously, Vercel's skew protection had some limitations and bugs that prevented it from working correctly across all of Astro's features. With this update, the Vercel adapter now uses the same new APIs to provide reliable skew protection when the `VERCEL_SKEW_PROTECTION_ENABLED` environment variable is enabled, bringing it to feature parity with Netlify.
+
+## Granular font preload filtering
+
+The experimental Fonts API now supports granular control over which font variants get preloaded. Previously, you could only choose to preload all variants of a font or none at all. Now you can specify exactly which weights, styles, and subsets should be preloaded, helping you optimize performance by downloading only what you actually use.
+
+When using the `<Font>` component, you can pass a `preload` array to filter variants:
+
+```astro
+---
+import { Font } from 'astro:assets';
+---
+
+<Font
+  cssVariable="--font-roboto"
+  preload={[
+    { subset: 'latin', style: 'normal' },
+    { weight: '400' },
+  ]}
+/>
+```
+
+This example will only preload the Latin subset in normal style, and any font files with weight 400. For variable weight fonts (which support a range like 100–900), Astro intelligently matches them against your filter criteria. Requesting weight `400` will include a variable font that covers the 100–900 range, since it supports that weight.
+
+You can filter on any combination of:
+- **`weight`** - Specific font weight (e.g., `'400'`, `'700'`)
+- **`style`** - Font style (e.g., `'normal'`, `'italic'`)
+- **`subset`** - Font subset (e.g., `'latin'`, `'cyrillic'`)
+
+This gives you fine-grained control over your font loading strategy, reducing unnecessary downloads while maintaining design flexibility.
+
+## Other improvements
+
+- **Copy button for stack traces** – The error overlay now includes a copy button to easily share stack traces with LLMs and AI editors for faster debugging. ([#14489](https://github.com/withastro/astro/pull/14489))
+- **Wrangler configuration scaffolding** – The `astro add` CLI now automatically scaffolds `wrangler.jsonc` when adding Cloudflare integration. ([#14564](https://github.com/withastro/astro/pull/14564))
+- **Node.js 18 deprecation** – The Vercel adapter now marks Node.js 18 as deprecated in favor of newer LTS versions. ([#14555](https://github.com/withastro/astro/pull/14555))
+
+## Bug fixes
+
+As always, we've been working hard on fixing issues since the [5.14 release](/blog/astro-5140). See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for all the details.
+
+## Community
+
+The Astro core team is:
+
+<Mention name="alex" />, <Mention name="ben" />, <Mention name="caleb" />,
```

Comment:
We will need to add the proper files so we can add Armand here! (I'll make a separate PR for that)